CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# SqlToCsvStream Changelog
+
+## 0.4.0 / unreleased
+
+### Features
+
+* 
+
+
+### Bug fixes
+
+* 
+
+
+## 0.3.0 / 2019-08-21
+
+### Security
+
+* upgrade nokogiri (addressed CVE-2019-5477)
+* upgrade all dependent gems
+* moved to CircleCI
```

Comment:
In all but the last bullet points, you use present tense… 🤓

Reply:
👍 fixed